src/ImageSharp/Formats/Bmp/BmpEncoderCore.cs

```diff
@@ -340,15 +339,15 @@ private void Write8BitColor<TPixel>(Stream stream, ImageFrame<TPixel> image, Spa
         {
             using IQuantizer<TPixel> frameQuantizer = this.quantizer.CreatePixelSpecificQuantizer<TPixel>(this.configuration);
             using IndexedImageFrame<TPixel> quantized = frameQuantizer.BuildPaletteAndQuantizeFrame(image, image.Bounds());
+            using IMemoryOwner<Rgba32> rgbColorsBuffer = this.memoryAllocator.Allocate<Rgba32>(quantized.Palette.Length);
+            Span<Rgba32> rgbColors = rgbColorsBuffer.GetSpan();
 
             ReadOnlySpan<TPixel> quantizedColors = quantized.Palette.Span;
-            var color = default(Rgba32);
+            PixelOperations<TPixel>.Instance.ToRgba32(Configuration.Default, quantizedColors, rgbColors);
 
-            // TODO: Use bulk conversion here for better perf
             int idx = 0;
-            foreach (TPixel quantizedColor in quantizedColors)
+            foreach (Rgba32 color in rgbColors)
```

Comment:
Why not something like:
```C#
PixelOperations<TPixel>.Instance.ToRgba32(Configuration.Default, quantizedColors, MemoryMarshal.Cast<byte, Rgba32>(colorPalette));
ref Span<uint> colorPaletteAsUInt= MemoryMarshal.Cast<byte, uint>(colorPalette);
for (int i =0;i<colorPaletteAsUInt;i++)
  colorPaletteAsUInt[i] = colorPaletteAsUInt[i] & 0xFFFFFF00;
```

Reply:
Tidy!

Reply:
yeah that will do, it just needs to be ToBgra instead

Reply:
Which should be fast enough with the new shuffle intrinsics